File: PHP/palindrome.php
<?php

$a = "kasur";
$b =  strrev($a);

    $string_reverse = str_split($b);
    $palin = '';

    foreach($string_reverse as $value){
        $palin.= $value; 
    }
    print $palin;

    if($a == $palin){
        print "<br>Palindrome";
    } else {
        print "<br>Not Palindrome"; 
    }
?>

File: C++/palindrome.cpp
#include <string>
using namespace std;

bool palindrome(string s){
    int length = s.length();
    for (int i = 0; i < length/2; ++i){
        if (s[i] != s[(length - 1) - i])
            return false;
    }
    return true;
}
